testGeogrid_ISCE.py

```diff
@@ -144,13 +144,14 @@ def loadMetadata(indir):
 
 def loadMetadataOptical(indir):
```

Comment:
In the main function, you now use **coregisterLoadMetadataOptical** for optical imagery. It seems this function **loadMetadataOptical** is deprecated. Do you still want to include it or just get rid of it?

Reply:
either way works for me -- I'd lean remove it.

Reply:
Removed